### PR TITLE
Update fine calo proc modifier

### PR DESCRIPTION
**Pull request by kdlong**
#### PR description:

Fix the use of the fineCalo procmodifer to properly set EMinFineTrack to 0, which is needed in master

#### PR validation:

Tested that the --fineCalo proc modifier works as intended when added to the cmsDriver command from 41034.202

Additional point: it would be good to have a workflow that tests fineCalo as well. Should I create one copying 41034.202, or another workflow?

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

File: SimG4Core/Application/python/g4SimHits_cfi.py [hunk 1]
```diff
@@ -65,7 +65,9 @@
 ## enable fine calorimeter functionality: must occur *before* common PSet is used below
 from Configuration.ProcessModifiers.fineCalo_cff import fineCalo
 fineCalo.toModify(common_MCtruth,
-    DoFineCalo = True
+    DoFineCalo = True,
+    UseFineCalo = [2],
+    EminFineTrack = cms.double(0.0),
```
Comment (perrotta) [inline SimG4Core/Application/python/g4SimHits_cfi.py:70]:
```suggestion
    EminFineTrack = 0.0
```

**Comment by kdlong** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**